synfig-core/src/modules/mod_noise/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ add_library(mod_noise MODULE
         "${CMAKE_CURRENT_LIST_DIR}/valuenode_random.cpp"
 )
 
-target_link_libraries(mod_noise synfig ${SIGCPP_LIBRARIES})
+target_link_libraries(mod_noise PRIVATE synfig)
```

Comment:
Why `PRIVATE` here, but not above with `mod_jpeg`, `mod_magickpp`?

Reply:
At first I forgot to fix it, but since I plan to return to this task, I left it as is.

Reply:
The main goal here was to get rid of the `include_directories` because they add include directories to all sub targets.

Reply:
> At first I forgot to fix it, but since I plan to return to this task, I left it as is.

So should I approve?

Reply:
Yes.

Reply:
Done